docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3.1'
+services:
+  etcd:
+    image: bitnami/etcd:3.4.3
+    volumes:
+      - .:/etcd-backup/
+    working_dir: /etcd-backup
+    environment:
+      - ALLOW_NONE_AUTHENTICATION=yes
+      - ETCDBACKUP_AWS_ACCESS_KEY=xxx
+      - ETCDBACKUP_AWS_SECRET_KEY=yyy
+      - ETCDBACKUP_PASSPHRASE=ciccio
+  prometheus:
+    image: prom/prometheus:v2.13.1
```

Comment:
I would probably aligned with the Prometheus repo and used our retagged images, but maybe for testing it's not necessary. Up to you

Reply:
it's used for local development only, I don't think there's need to pollute our docker registries for that.